src/components/index.scss

```diff
@@ -41,3 +41,4 @@
 @import "./AutoSuggestion/AutoSuggestion";
 @import "./Badge/badge";
 @import "./PropertyValuePair/propertyvalue";
+@import "./MultiSelect/multiselect";
```

Comment:
If you create a new stylesheet then please do it with correct folder and file name, in this case it would be `_multisuggestfield.scss` in `MultiSuggestField` folder. `MultiSelect` is the deprecate component name, the JS code only lives still in the old folder to keep author information.